### ISSUE TITLE
[Quest] Green Technology

### ISSUE THREAD

**Issue by Taehupo** [dalaranwow/dalaran-wow#4729]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Fringe Engineer Tezzla is absent from where she'd need to be to return the quest to her
**Expected behaviour**: Tell us what should happen instead
She should be on the platform amongst the other goblins to be able to return the quest
**Steps to reproduce the problem**:

1. Take the Green Technology quest from Orgrimm's Hammer
2. Go to the platform where the Fringe Engineer Tezzla should be to giver the quest back
3. She isn't amongst the NPCs

**Include proofs for this behaviour**
http://www.wowhead.com/quest=13379/green-technology

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Fringe Engineer Tezzla ID : 32430
![wowscrnshot_053117_160335](https://cloud.githubusercontent.com/assets/10206835/26635827/46ae4e18-461b-11e7-8ffd-1566a1a58363.jpg)

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**
http://puu.sh/w6Aib/fd625630b9.jpg

**Comment by Rushor** [comment]
duplicate of https://github.com/dalaranwow/dalaran-wow/issues/759
